### PR TITLE
Update issue template with multiple labels

### PR DESCRIPTION
**Pull request by VachaShah**
Signed-off-by: Vacha Shah <vachshah@amazon.com>

### Description
This PR intends to add multiple labels for the issue template to better categorize the issues.
 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [X] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_template.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 name: 🐛 Bug report
 about: Create a report to help us improve
 title: "[BUG]"
-labels: bug
+labels: 'bug, untriaged, Beta'
```
Comment (mihirsoni) [inline .github/ISSUE_TEMPLATE/bug_template.md:5]: Should we add tag beta by default ?

**Review by mihirsoni — COMMENTED** [review]
@VachaShah  Thanks for the PR, one small comment otherwise LGTM !

**Review by mihirsoni — APPROVED** [review]
LGTM !! Thanks for the changes @VachaShah

**Review by boktorbb-amzn — APPROVED** [review]
LGTM

File: .github/ISSUE_TEMPLATE/bug_template.md (hunk 1 above)
Comment (mihirsoni) [reply .github/ISSUE_TEMPLATE/bug_template.md:5]: Will it mean, we'll change this `beta` tag for after each release?

**Pull request merged**